notebook/API.livemd

```diff
@@ -11,8 +11,107 @@ base_url = IO.gets("Base URL") |> String.trim()
 ## Users
 
 ### GET /v1/me
+
 Used to fetch the current user's profile details
 
+### 200 OK
+
+```elixir
+headers = [access_token: "77564df8-7823-45b2-b9ef-38dac927ddfd"]
+
+LivemanApi.get(base_url, "/v1/me", %{}, headers)
+```
+
+## Surveys
+
+### GET /v1/surveys
+
+Used to fetch all the available surveys.
+
+Returns a 200 OK response
+
 ```elixir
-LivemanApi.get(base_url, "/v1/me")
+LivemanApi.get(base_url, "/v1/surveys")
+```
+
+Supports pagination by passing the page number and page size as params
+
+```elixir
+LivemanApi.get(base_url, "/v1/surveys", %{page: 2, page_size: 8})
+```
+
+### POST /v1/respnoses
+
+Used to create an answer to a particular survey
+
+Requires a valid access_token in the header
+
+Returns a 201 Created response
+
+#### 201 OK - Valid Params
```

Comment:
`OK` is used for `200`, so I think it should be `Created` https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html
```suggestion
#### 201 Created - Valid Params
```

Reply:
Nice catch! 👍